django/bossingest/views.py

```diff
@@ -226,60 +226,62 @@ def post(self, request):
 
         """
         ingest_config_data = request.data
-
-        # Add metrics to CloudWatch
-        extent = ingest_config_data['ingest_job']['extent']
-        tile_size = ingest_config_data['ingest_job']['tile_size']
-        database = ingest_config_data['database']
-
```

Comment:
Alternatively, we could just run this part only if the ingest job is a tile ingest.

Reply:
I am going to add another ticket to test this more thoroughly.